src/Cairo.jl

```diff
@@ -30,14 +30,23 @@ end
 
 include("constants.jl")
 
+# check library version, error message if too low
+cairo_version = ccall((:cairo_version,Cairo._jl_libcairo),Int32,())
+
+if cairo_version < 11200
+    error("cairo version " * string(cairo_version) * ", less then 1.12.00")
```

Comment:
always erroring at package import time doesn't seem necessary if only the new functionality requires recent versions?